src/app/settings/settings.effects.spec.ts

```diff
@@ -73,16 +63,4 @@ describe('SettingsEffects', () => {
       );
     });
   });
-
-  it('should not call methods on AnimationsService and LocalStorageService for other actions', () => {
```

Comment:
I think this test is removed by accident? Or was this intended?

Reply:
@zetsnotdead pls have a look :)

Reply:
I actually deleted it deliberately. My reason being two fold:
- no other effect spec tests for actions that do nothing, so this test seemed a little out of place
- if we tests for all actions that trigger an event, can't we just assume that all other actions are a noop

If this reasoning does not seem right I will revert it.

Reply:
I think I have the same mindset as @zetsnotdead here 😄 
Especially on the 2nd point.

Reply:
@zetsnotdead @timdeschryver  thanks for clarification!